v3/src/models/history/apply-model-change.ts

```diff
@@ -13,6 +13,7 @@ export interface INotification {
 export interface IApplyModelChangeOptions {
   log?: string | ILogMessage | (() => Maybe<string | ILogMessage>)
   notify?: INotification | INotification[] | (() => Maybe<INotification | INotification[]>)
+  webViewId?: string
```

Comment:
```suggestion
  notifyTileId?: string
```
From the standpoint of this function, it doesn't matter whether it's a web view. It's the tile that should be notified when we're broadcasting to a single tile rather than all tiles.